app/helpers/file_helper.rb

```diff
@@ -2,9 +2,12 @@ module FileHelper
   def syntax_highlighter_for_filename(filename, track)
     parts = filename.split(".")
 
-    lang = parts.size > 1 ?
-      Exercism::PrismFileMappings[parts.last.downcase] :
-      track.syntax_highligher_language
+    if !parts.empty?
```

Comment:
I'd suggest `if parts.present?` rather than checking for the negative of the negative.